app/helpers/middleware_server_helper/textual_summary.rb

```diff
@@ -0,0 +1,55 @@
+module MiddlewareServerHelper::TextualSummary
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    @prop_hash = eval(@record.properties)
```

Comment:
I really don't like this eval.  If it's a properly serialized column, this should not be necessary.